core/chains/evm/forwarders/orm_test.go

```diff
@@ -5,17 +5,17 @@ import (
 	"errors"
 	"testing"
 
+	"github.com/smartcontractkit/chainlink-common/pkg/sqlutil"
+
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/jmoiron/sqlx"
+
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/utils/big"
 	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils"
 	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils/pgtest"
-	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
-
-	"github.com/jmoiron/sqlx"
 )
 
 type TestORM struct {
```

Comment:
Since we're replacing the db in the actual struct with a `sqlutil` interface, is it possible to replace the db in the `TestORM` with something so we can lose the `sqlx` import? `sqlutil.Queryer` wouldn't work would it?

Reply:
Yup, we can use `sqlutil.queryer` here as well!